modules/tinymce/src/core/main/ts/fmt/MergeFormats.ts

```diff
@@ -110,23 +104,22 @@ const unwrapEmptySpan = function (dom: DOMUtils, node: Node) {
   }
 };
 
-const processUnderlineAndColor = function (dom: DOMUtils, node: Node) {
-  let textDecoration;
-  if (node.nodeType === 1 && node.parentNode && node.parentNode.nodeType === 1) {
-    textDecoration = FormatUtils.getTextDecoration(dom, node.parentNode);
-    if (dom.getStyle(node, 'color') && textDecoration) {
-      dom.setStyle(node, 'text-decoration', textDecoration);
-    } else if (dom.getStyle(node, 'text-decoration') === textDecoration) {
-      dom.setStyle(node, 'text-decoration', null);
+const mergeUnderlineAndColor = function (dom: DOMUtils, format, vars: FormatVars, node: Node) {
+  const processUnderlineAndColor = function (n: Node) {
+    if (n.nodeType === 1 && n.parentNode && n.parentNode.nodeType === 1) {
+      const textDecoration = FormatUtils.getTextDecoration(dom, n.parentNode);
+      if (dom.getStyle(n, 'color') && textDecoration) {
+        dom.setStyle(n, 'text-decoration', textDecoration);
+      } else if (dom.getStyle(n, 'text-decoration') === textDecoration) {
+        dom.setStyle(n, 'text-decoration', null);
```

Comment:
In this change, `processUnderlineAndColor` is enclosed within its only caller `mergeUnderlineAndColor`, replacing the curry with simple closure.

Reply:
it's easier to follow this with "hide whitespace changes" 😁

Reply:
I think that applies for all PRs :)